python/cudf/cudf/_lib/cpp/libcpp/optional.pxd

```diff
@@ -0,0 +1,50 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023, NVIDIA CORPORATION &
+# AFFILIATES. All rights reserved.  SPDX-License-Identifier:
+# Apache-2.0
```

Comment:
Do SPDX identifiers work correctly (as machine-readable license information) if they are broken across lines? Probably no, but also not something I want to block on. Just noting it.